analytical/templatetags/google_analytics_gtag.py

```diff
@@ -72,6 +75,24 @@ def render(self, context):
             html = disable_html(html, 'Google Analytics')
         return html
 
+    def _get_event_commands(self, context):
+        values = (
+            context.get('google_analytics_event%s' % i) for i in range(1, 6)
+        )
+        params = [(i, v) for i, v in enumerate(values, 1) if v is not None]
+        commands = []
+        for _, var in params:
+            key, value = var
+            try:
+                dict(value)
+            except ValueError:
+                value = f"'{value}'"
```

Comment:
This doesn't look like a valid pattern. The `try`-block shouldn't be use for testing on a data type. And what's code in the `except`-clause? I think this is an edge case of a "pythonic try-except" where the pattern doesn't fit.

It's not clear at first sight what we're trying to achieve. Maybe an `if instanceof(value, dict)` would be better here, because it's more explicit for the use case.